src/ducc0/fft/fft.h

```diff
@@ -161,11 +161,6 @@ struct util // hack to avoid duplicate symbols
         "axis length mismatch");
     }
 
-#ifdef DUCC0_NO_THREADING
```

Comment:
This is removed to allow us to define `DUCC0_NO_THREADING` (and exclude the use of `std::mutex`/`std::thread`) while still allowing our custom threadpool to be given a non-unit thread count.  The `Distribution` class in threading.cc will limit the total number of threads to 1 anyways if `DUCC0_NO_THREADING` is defined.